R/students.R

```diff
@@ -1446,11 +1446,16 @@ s_33 <- function(input_df=usheUtils::fake_student_df) {
 }
 
 
-#' Calculate USHE Element s_34 (SSID aka USOE Unique ID)
+#' Calculate USHE Element s_34, g_20 (SSID aka USOE Unique ID)
```

Comment:
@cdemkegit Why is g_20 listed but not implemented in this function?

Reply:
@cdemkegit 
I'm going to merge, regardless. Since this is just documentation and doesn't break anything.

I imagine this is going to be a general function in the next update?